src/services/mentees.js

```diff
@@ -1504,6 +1521,92 @@ module.exports = class MenteesHelper {
 			return error
 		}
 	}
+
+	/**
+	 * Retrieves a communication token for the logged in user.
+	 *
+	 * This asynchronous method logs in a user using their unique identifier (`id`)
+	 * to obtain a communication token and other relevant details, then returns a
+	 * standardized success response with the token, user ID, and metadata.
+	 *
+	 * @async
+	 * @function getCommunicationToken
+	 * @param {string} id - The unique identifier of the user for whom the communication token is to be retrieved.
+	 * @returns {Promise<Object>} A promise that resolves to an object containing the response code, message, result data,
+	 * and additional metadata.
+	 *
+	 * @throws {Error} If the communicationHelper login process fails, this method may throw an error.
+	 *
+	 * @example
+	 * const response = await getCommunicationToken(123);
+	 * console.log(response);
+	 * // {
+	 * //   responseCode: "OK",
+	 * //   message: "Communication token fetched successfully!",
+	 * //   result: {
+	 * //     auth_token: "_GTFHENH422lGlLcgYQfu2GnnWO8bg6zY8ZHrXkcNmN",
+	 * //     user_id: "Q9hz3jbPXkk3fXQoL"
+	 * //   },
+	 * //   meta: {
+	 * //     correlation: "69893cb9-8b0c-44f9-945e-1bff2174af0d",
+	 * //     meetingPlatform: "BBB"
+	 * //   }
+	 * // }
+	 */
+	static async getCommunicationToken(id) {
```

Comment:
@nevil-mathew for mentor and mentee or if any other roles we may need API. you can move respected ocntroller.

Reply:
ok